### PR TITLE
[Refactoring] Move remaining test classes in mailbox-elasticsearch to JUnit 5

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java [hunk 1]
```diff
@@ -46,36 +46,35 @@
...
 
-    private TextExtractor textExtractor;
+    TextExtractor textExtractor;
+    Date date;
+    PropertyBuilder propertyBuilder;
 
-    private Date date;
-    private PropertyBuilder propertyBuilder;
+    @RegisterExtension
+    static TikaExtension tika = new TikaExtension();
```
Comment (trantienduchn) [inline mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java]: does `TikaExtension` hold a singleton docker tika?

Comment (Arsnael) [reply mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java]: Nope... good point :)

Comment (Arsnael) [reply mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java]: The fix is related to an other PR though, where I introduced `TikaExtension` (see #2964) in this commit : 
https://github.com/linagora/james-project/pull/2964/commits/d8b7fec43b2e9c6af06d9d5690bb52dd4803415a

**Comment by Arsnael** [comment]
The commits related to test classes using Tika container have been moved to #2976

**Comment by chibenwa** [comment]
Was :green_apple:

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**